### DISCUSSION TITLE
显示撤回消息了但群里还有消息咋回事啊

### DISCUSSION THREAD

**Discussion by XiaoFuOS** [Mrs4s/go-cqhttp#1900] [Q&A]
在后台日志里显示消息撤回了，但是群里的消息没有撤回，啥原因啊？不知道怎么解决🤔

**Comment by Xiaoxusheng** [comment]
肯定是你的逻辑有问题，我可以撤回

**Reply by XiaoFuOS** [reply]
> 肯定是你的逻辑有问题，我可以撤回

解决了老哥，是我手机qq的原因，电脑看是撤回了，手机还在，然后我切其他号切回来就正常了，只是手机qq显示的原因

**Reply by Xiaoxusheng** [reply]
解决了就好
